maud_extras/Cargo.toml

```diff
@@ -9,8 +9,8 @@ repository = "https://github.com/lfairy/maud_extras"
 description = "Extra bits and pieces for use in Maud templates."
 
 [dependencies]
-maud = { path = "../maud", features = ["iron"] }
-maud_macros = { path = "../maud_macros" }
+maud = { version = "0.14.0", features = ["iron"] }
```

Comment:
I agree that this is a bad idea, I wouldn't fix it like this, but fix it by freezing the rust version in the travis build